docs/source/user_guide.rst

```diff
@@ -430,40 +430,5 @@ And these are the results::
 
 .. _`Going to Mars with Python using poliastro`: http://nbviewer.ipython.org/github/poliastro/poliastro/blob/master/examples/Going%20to%20Mars%20with%20Python%20using%20poliastro.ipynb
 
-Working with NEOs
```

Comment:
Instead of removing all this, can we document how to use the alternatives, either `Orbit.from_sbdb` or `Orbit.from_horizons`?

Reply:
For sure! I will have it documented

Reply:
The heading must be changed no? As the horizons is meant for bodies in solar system. Its no more Near Earth?